guides/common/modules/proc_enabling-the-satellite-tools-repository.adoc

```diff
@@ -2,8 +2,15 @@
 = Enabling the {project-client-name} Repository
 
 The {project-client-name} repository provides the `katello-agent`, `katello-host-tools`, and `puppet` packages for clients registered to {ProjectServer}.
+It is recommended to enable the repositories for all {RHEL} versions to manage the hosts without any issues.
```

Comment:
Can someone please confirm this recommendation? Why not recommend "only enable/sync repos for OS you actually need/use"?

Reply:
> Additionally, the user should enable the repo for each RHEL version, which the hosts he intends to manage may have. You should mention this.

Addressing this comment from @Lennonka here.

Reply:
I actually meant the same thing as Max suggests. How about:
```suggestion
It is recommended to enable the repositories for all {RHEL} versions that you need to manage hosts.
```

Reply:
Addressed :+1: